src/samplereader/FragmentedSampleReader.cpp

```diff
@@ -261,6 +266,12 @@ bool CFragmentedSampleReader::GetInformation(kodi::addon::InputstreamInfo& info)
 
   isChanged |= m_codecHandler->GetInformation(info);
 
+  // store dvcc metadata
+  if (auto dvcc = ParseDvcc())
+  {
+    PopulateDvccMetadata(info, dvcc);
+  }
```

Comment:
when an info will be changed `isChanged` must be set to true, otherwise there is the risk that these new info will be not updated on kodi side

if you dont want to compare each DV info at least check if previously this info was missing, if so, and if you have this info to set, set true isChanged